README.md

```diff
@@ -23,7 +23,7 @@ request to Vercel. (We use Vercel because of their nice GitHub integration.)
 ## History
 
 This is a rewrite of the Deno website it will combine the code in
-https://github.com/denoland/deno/tree/master/website and
+https://github.com/denoland/deno/tree/master and
```

Comment:
It is better to use the available old tree instead:

```diff
- https://github.com/denoland/deno/tree/master/website
+ https://github.com/denoland/deno/tree/f96aaa802b245c8b3aeb5d57b031f8a55bb07de2
```

Reply:
thanks i change your suggestion  :) !